waku/factory/node_factory.nim

```diff
@@ -402,18 +402,12 @@ proc startNode*(
 
   # Connect to configured static nodes
   if conf.staticnodes.len > 0:
-    if not conf.relay:
-      return err("waku relay (--relay=true) should be set when configuring staticnodes")
     try:
       await connectToNodes(node, conf.staticnodes, "static")
     except CatchableError:
       return err("failed to connect to static nodes: " & getCurrentExceptionMsg())
 
   if dynamicBootstrapNodes.len > 0:
-    if not conf.relay:
-      return err(
-        "waku relay (--relay=true) should be set when configuring dynamicBootstrapNodes"
-      )
```

Comment:
We don't need these conditions that I added. When I added them I thought that `connectToNodes`, which called `connectRelay`, were only related to Relay protocol but the function name was a misnomer